src/JSON3.jl

```diff
@@ -98,7 +98,7 @@ function Base.iterate(arr::Array{T}, (i, tapeidx)=(1, 3)) where {T}
     i > length(arr) && return nothing
     tape = gettape(arr)
     @inbounds t = tape[tapeidx]
-    val = getvalue(T, getbuf(arr), tape, tapeidx, t)
+    val = getvalue(Any, getbuf(arr), tape, tapeidx, t)
```

Comment:
this is bad; this is how we can get fast, type-stable code when accessing elements of `JSON3.Array{T}`